data/json/furniture_and_terrain/furniture-industrial.json

```diff
@@ -481,7 +481,7 @@
       "sound_fail": "crash!",
       "items": [
         { "item": "scrap", "count": [ 1, 4 ] },
-        { "item": "cable", "count": [ 1, 8 ] },
+        { "item": "cable", "charges": [ 100, 800 ] },
```

Comment:
this is explicitly described as huge cylinders with copper wire so I left abundant wire drops from smashing